code/modules/atmospherics/machinery/portable/scrubber.dm

```diff
@@ -31,9 +31,9 @@
 		/datum/gas/halon,
 	)
 
-/obj/machinery/portable_atmospherics/scrubber/Destroy()
-	var/turf/T = get_turf(src)
-	T.assume_air(air_contents)
+/obj/machinery/portable_atmospherics/pump/on_deconstruction(disassembled)
```

Comment:
Scrubber -> pump?

Reply:
Oh whoops!

Reply:
I also realize that I should still keep a Destroy() for the atmos stuff, to at least, null the air contents.

Reply:
Oh wait no, that is already handled in the parent.